shell/common/common_benchmarks.sh

```diff
@@ -165,8 +165,52 @@ get_options() {
 
 # Temple functions, re implement in benchmark if needed
 
-benchmark_suite_config() {
-  logger "DEBUG: No specific ${FUNCNAME[0]} defined for $BENCH_SUITE"
+benchmark_suite_start_config() {
+  logger "INFO: preparing confiuration for $BENCH_SUITE"
+
+  #Java is always required regardless of the benchmark to run
+  source_file "$ALOJA_REPO_PATH/shell/common/common_java.sh"
+  set_java_requires
+
+  if [ ! -z "$use_hadoop" ]; then
```

Comment:
Why `! -z` rather than `-n`, or even just nothing at all `if [[ $use_hadoop ]]`? But this only checks whether the variable is set and non-empty, what if someone wrote `use_hadoop=false`?

Reply:
Indeed, changing the conditionals.